spring-data-jdbc/src/main/java/org/springframework/data/jdbc/core/AggregateChangeExecutor.java

```diff
@@ -37,11 +39,13 @@ class AggregateChangeExecutor {
 
 	private final JdbcConverter converter;
 	private final DataAccessStrategy accessStrategy;
+	private final JdbcExceptionTranslator jdbcExceptionTranslator;
```

Comment:
This approach lacks configurability.

Reply:
@mp911de The suggestion is to use `PersistentExceptionTrasnlator` interface here, right?